hack/jenkins/build.sh

```diff
@@ -33,6 +33,12 @@ export HOME=${WORKSPACE} # Nothing should want Jenkins $HOME
 export PATH=$PATH:/usr/local/go/bin
 export KUBE_SKIP_CONFIRMATIONS=y
 
+# Configure a temprary GOPATH.  The various go tools require this.
+mkdir -p .gopath/src/k8s.io
```

Comment:
possibly. it would also mean that all CI builds would have a -dirty in their version, which is unfortunate.